build-system/tasks/helpers.js

```diff
@@ -773,6 +777,7 @@ module.exports = {
   compileAllUnminifiedTargets,
   compileJs,
   compileTs,
+  BABELIFY_GLOBAL_TRANSFORM,
```

Comment:
Nit: Alphabetize.

Reply:
Done in 9b20dde42e6de814da71363c75ae9aeae4db8057